templates/default/hdfs-site.xml.erb

```diff
@@ -646,4 +716,24 @@
     </description>
   </property>
 
+  <property>
+    <name>dfs.cloud.block.report.delay</name>
+    <value>3600000</value>
+    <description>
+    </description>
+  </property>
+
+  <property>
+    <name>dfs.cloud.mark.blocks.corrupt.or.missing.after</name>
+    <value>3600000</value>
+    <description>
+    </description>
+  </property>
+
+  <property>
+    <name>dfs.cloud.max.br.threads</name>
+    <value>10</value>
+    <description>
+    </description>
+  </property>
```

Comment:
can you add these as attributes as well?

Reply:
did it